src/Layers/xrRender/Blender_detail_still.cpp

```diff
@@ -89,10 +89,10 @@ void CBlender_Detail_Still::Compile(CBlender_Compile& C)
     switch (C.iElement)
     {
     case SE_R2_NORMAL_HQ: // deffer wave
-        uber_deffer(C, false, "detail_w", "base", true, nullptr, false, C._defSubPath);
+        uber_deffer(C, false, "detail_w", "base", true, nullptr, false, C.ShadersSubPath);
```

Comment:
Please don't mix adding the resources and editing of the code in the same commit. Split it to different commits.